### ISSUE TITLE
Query regarding wash Kubernetes manifests.

### ISSUE THREAD

**Issue by r-aju** [wazuh/wazuh-kubernetes#309]
Heyy,

I'm currently following the official [wazuh-kubernetes](https://github.com/wazuh/wazuh-kubernetes) guide and deploying server via Kubernetes manifests.

While deploying the [wazuh_manager](https://github.com/wazuh/wazuh-kubernetes/tree/master/wazuh/wazuh_managers) stack found out that one of the port 1514 is not added for the agents to communicate with the server in [loadbalancer](https://github.com/wazuh/wazuh-kubernetes/blob/master/wazuh/wazuh_managers/wazuh-master-svc.yaml) config file.

As per the official document of wazuh it says that we'll need to open port 1514 also, which is missing.
![image](https://user-images.githubusercontent.com/43216503/200309453-a23b933f-3b21-48e5-839b-8c881791798f.png)

Ideally we'll have to add port 1514 as well like this in the [Loadbalancer](https://github.com/wazuh/wazuh-kubernetes/blob/master/wazuh/wazuh_managers/wazuh-master-svc.yaml) config file.

  ports:
    **- name: agent-communication
      port:  1514
      targetPort: 1514**
    - name: registration
      port: 1515
      targetPort: 1515
    - name: api
      port: 55000
      targetPort: 55000

**Comment by r-aju** [comment]
Tagging @vcerenu, @alberpilot and @xr09 since, you guy's contributed to this previously !!

**Comment by r-aju** [comment]
@vcerenu , Yes this is enabled on Wazuh manager worker LoadBalancer. But, can this also be enabled on Wazuh Manager Master's LB as well.

It got working in my case when I added it in manager's LB.

**Comment by vcerenu** [comment]
Yes, that configuration works but it is not the recommended one. The registration of new agents (port 1515) and the communication with the API (port 55000) is carried out with the `master` node only, but the sending of data from the agent to the manager (port 1514) can be carried out towards any node, for which the availability of the `master` node is maintained with the tasks that only he can perform only and LoadBalancer is enabled with port 1514 for sending logs to all other `worker `nodes of the cluster.

Your configuration works, but I would only enable it if I am not going to use a cluster of Wazuh managers with a master node and worker nodes.

**Comment by r-aju** [comment]
Got it, Thanks for the clear explanation about the various sub components.
